tests/fixtures/sample-with-diagram.adoc

```diff
@@ -14,3 +14,119 @@ Thanks to link:https://github.com/ryosms[]!
 class Sample {
 }
 ----
+
+
+= Additional Diagrams
+
+== Dita Example
+[ditaa]
+....
+                   +-------------+
+                   | Asciidoctor |-------+
+                   |   diagram   |       |
+                   +-------------+       | PNG out
+                       ^                 |
+                       | ditaa in        |
+                       |                 v
+ +--------+   +--------+----+    /---------------\
+ |        | --+ Asciidoctor +--> |               |
+ |  Text  |   +-------------+    |   Beautiful   |
+ |Document|   |   !magic!   |    |    Output     |
+ |     {d}|   |             |    |               |
+ +---+----+   +-------------+    \---------------/
+     :                                   ^
+     |          Lots of work             |
+     +-----------------------------------+
+....
+
+
+== Graphviz ( Digraph)
+[graphviz, dot-example, svg]
+----
+digraph g {
+    a -> b
+    b -> c
+    c -> d
+    d -> a
+}
+----
+
+== Plantuml from external file...
+
+plantuml::activity_diagram.txt[format="svg", align="center"]
```

Comment:
It seems there is a bug on the macro that hit us here (relative path is resolved from the caller and not from the document): https://github.com/asciidoctor/asciidoctor-diagram/issues/173

Can you use the block + include syntax here, since it is not related to the docker image ?

Here is a sample I tried locally with success:
```
[plantuml,activity,svg,align="center"]
----
include::activity_diagram.txt[]
----
```

Reply:
Ok - will do.